### ISSUE TITLE
[Keyboard Navigation - WIX - License Term] Links inside "Microsoft software License Term" section are not accessible via keyboard.

### ISSUE THREAD

**Issue by sikoppar** [wixtoolset/issues#6040]
* Which version of WiX are you building with?
> 3.7.1224.1

* Which version of Visual Studio are you building with (if any)?
> Visual Studio 2017 (v141)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
>0.9.21.62588

* Which version of .NET are you building with?
> Not Applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
>Not Applicable

* Describe the problem and the steps to reproduce it.
User Experience:

If, any control is not accessible by the keyboard, then the users who depends upon keyboard for navigation might get affected and may not be able to use the complete functionality of the application.

Pre Requist:
1. Download the installer from https://www.microsoft.com/en-us/download/details.aspx?id=54688
2. Navigate to NPS Extension for Azure MFA Setup and activate it by using Enter key.

Repro Steps:
1. Navigate all the controls of "NPS Extension for Azure MFA " popup window.

Actual Result:
Keyboard focus does not move to the links which is inside  "Microsoft software Licence Term" section . Hence Links inside "Microsoft software License Term" section are not accessible via keyboard.

Expected Result:
Links inside "Microsoft software License Term" section should be accessible via keyboard.

**Comment by BMurri** [comment]
That version of WiX is over 5 years old, and there have been changes to the WixStdBa since then. Do these issues replicate with v3.11.1?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by barnson** [comment]
This issue is open and unassigned. That means it's waiting for someone to investigate the root problem, discuss possible solutions to the problem, and implement an agreed solution.

If you are interested in doing that work yourself, our [developer documentation](https://docs.firegiant.com/wix/development/) provides a great checklist for getting started.

If you are not interested in doing that work, someone else needs to become interested enough to do it. If this issue has been open for a long time, there probably isn't general interest in this particular issue. In that case, you'll want to consider how to motivate others to fix it for you. This is a pretty [good list of support options](https://www.firegiant.com/support-options/).
